docs/getting-started/download.md

```diff
@@ -1,8 +1,30 @@
 ---
-sidebar_position: 2
+sidebar_position: 1
 ---
 
-# Installation
+# Download
+
+## Bootstrap Script
+
+You can use the bootstrap script to download TigerBeetle:
+
+### macOS/Linux
+
+```shell
+git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle && ./bootstrap.sh
+```
+
+### Windows
+
+```powershell
+git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle && .\bootstrap.ps1
+```
+
+### Build from Source
+
+To build TigerBeetle from source, add `-build` as an argument to the bootstrap script.
```

Comment:
It's worth mentioning here that the user may want to build from sources when the target CPU is too old and lacks some of the instructions the prebuilt binary requires (i.e. aes).